esp_schedule/README.md

```diff
@@ -0,0 +1,50 @@
+# ESP Scheduling
+
+[![Component Registry](https://components.espressif.com/components/espressif/esp_schedule/badge.svg)](https://components.espressif.com/components/espressif/esp_schedule)
+
+This component is used internally by ESP RainMaker to implement schedules.
+
+> Note: By default, the time is w.r.t. UTC. If the timezone has been set, then the time is w.r.t. the specified timezone.
+
+## Test code:
+
+```
+#include <esp_schedule.h>
```

Comment:
```suggestion
#include "esp_schedule.h"
```

By convention, angle brackets are used for system and C or C++ standard library header files.